### PR TITLE
[FEATURE ember-improved-instrumentation] Basic actions

### PR DESCRIPTION
**Pull request by asakusuma**
Ensure basic actions, not just closure actions, are instrumented

### CODE REVIEW

File: packages/ember-glimmer/tests/integration/helpers/element-action-test.js [hunk 1]
```diff
@@ -25,6 +27,54 @@ function getActionAttributes(element) {
...
+          foo() {}
+        }
+      });
+
+      this.registerComponent('example-component', {
+        ComponentClass: ExampleComponent,
+        template: '<button {{action "foo" "bar"}}>Click me</button>'
+      });
+
+      let subscriber = subscribe('interaction.ember-action', {
```
Comment (rwjblue) [inline packages/ember-glimmer/tests/integration/helpers/element-action-test.js]: I think this and the teardown should be done in module setup/teardown to ensure we don't leak subscribers if this errors.

File: packages/ember-glimmer/tests/integration/helpers/element-action-test.js [hunk 2]
```diff
@@ -25,6 +27,54 @@ function getActionAttributes(element) {
...
+
+      this.runTask(() => {
+        this.$('button').click();
+      });
+
+      this.assert.equal(subscriberCallCount, 1, 'subscriber has been called 1 time');
+      this.assert.equal(subscriberPayload.name, 'foo', 'subscriber called with correct name');
+      this.assert.equal(subscriberPayload.args[0], 'bar', 'subscriber called with correct args');
+
+      unsubscribe(subscriber);
```
Comment (rwjblue) [inline packages/ember-glimmer/tests/integration/helpers/element-action-test.js]: Can you move to modules `afterEach` / `teardown`?

**Comment by asakusuma** [comment]
@rwjblue moved to `reset` instead of `unsubscribe`, and moved it to `teardown`. Can you start travis? For whatever reason it's not starting on it's own.

**Comment by rwjblue** [comment]
Ya, Travis has been taking a few minutes to schedule a job (and set the "pending" commit status).  Looks like it is running now.

**Comment by asakusuma** [comment]
Ah I see. Did not realize it could be that async. Looks like things are green.

**Pull request merged**